src/components/atoms/List/List.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { List as AntList, ListProps as AntListProps } from 'antd'
+
+import { PreviewProps } from '@/types/preview'
+import { ComponentProps } from '@/types/component'
+
+export interface ListProps<T> extends PreviewProps, ComponentProps<AntListProps<T>> {}
+
+const List = React.forwardRef<HTMLDivElement, ListProps<any>>((props, ref) => {
```

Comment:
T here?

Reply:
@quy-truong it because AntListProps but i will remove it too